js/ui/overlay.js

```diff
@@ -413,6 +413,14 @@ var Overlay = Widget.inherit({
                     }
                 }
             }
+        }, {
+            device: function() {
+                return !windowUtils.hasWindow();
```

Comment:
We do not render overlays on a server, so it is not necessary to set the default option for this case.

Reply:
We should setup an initial size on markup initialization. I've change defaults for the server side rendering